src/program-common/eventhivtransmission.cpp

```diff
@@ -172,6 +176,19 @@ double EventHIVTransmission::solveForRealTimeInterval(const State *pState, doubl
 	return Tdiff/h;
 }
 
+int EventHIVTransmission::getH(const Person *pPerson)
+	{
+		assert(pPerson != 0);
+      
+     		bool H1 = pPerson->hsv2().isInfected();
+
+     		int H = 0;
+     		if (H1 == true)
+         	H = 1;
+     		return H;
+
+	} 
+
```

Comment:
Please remove the initial indentation to get a more similar code style as the rest of the code. Similarly, remove the unnecessary empty line before the last closing curly brace